Makefile_services.mk

```diff
@@ -48,6 +48,22 @@ service-ioc-as-ti-start:
 		--extra-vars "ctrl_service_state=started" \
 		-u sirius -k --ask-become-pass playbooks-services/playbook-service-ioc-as-ti.yml
 
+# === BL AP IOC services ===
```

Comment:
same naming issue applied here, @anacso17

Reply:
how should we name this ioc group?

Reply:
maybe `si-ap-bl`?